packages/x-charts/src/ChartsSurface/ChartsSurface.tsx

```diff
@@ -18,6 +18,7 @@ export interface ChartsSurfaceProps {
   desc?: string;
   sx?: SxProps<Theme>;
   children?: React.ReactNode;
+  tabIndex?: number;
```

Comment:
Should `ChartsSurfaceProps` extend `React.ComponentProps<'svg'>`? Then we wouldn't need to wait for users to complain so we can add more attributes to the types. Same goes for the legend.

Reply:
We could, though that is generally bad for prop discoverability... At least on VSCode 😢
@alexfauquette do you have any input?